www/student.html

```diff
@@ -19,7 +19,7 @@ <h1>Input:</h1>
       </div>
       <div id="output-displays">
         <button id="run-button">Run</button>
-        <button id="end-button" disabled>End</button>
+        <button id="end-button" disabled="true">End</button>
```

Comment:
Boolean attributes should not be set to "true" in HTML5. They are `true` if they are present and `false` otherwise.

More info: https://developer.mozilla.org/en-US/docs/Glossary/Boolean/HTML